tests/documentation/test_entrypoints.py

```diff
@@ -0,0 +1,23 @@
+import os
+import pytest
+
+
+class TestEntryPoints:
+
+    def test_entry_points(self):
+        parent_dir = os.pardir
+        readme_path = os.path.join(os.path.dirname((parent_dir)), 'supportedextensions.md')
+        with open(readme_path, 'r') as file:
+            readme_ext = [line.strip()[2:] for line in file if '*' in line]
+        setup_path = os.path.join(os.path.dirname((parent_dir)), 'setup.py')
+        with open(setup_path, 'r') as file:
+            parse = False
+            setup_ext = []
+            for line in file:
+                if parse and '#' not in line and ' = ' in line:
+                    setup_ext.append(line.strip().split(' = ')[0][1:])
+                # after you see this line its okay to start parsing
+                if '\'mfr.renderers\': [' in line:
```

Comment:
- [ ] To make this more robust and more efficient, we can do equality check on the trimmed line.

Reply:
Here is an example of a more efficient `if` statement:

```python
for line in file:
    line = line.strip()
    if parse:
        if line.startswith('#') and ' = ' in line:
            ...
    elif line == '\'mfr.renderers\': ['
        ...
```

Reply:
Changed test to completely get rid of If statements. this makes it much cleaner and nicer.
Found out how to get access to the pkg_resources from a PR @Johnetordoff  put in (thanks John!). Had googled for it earlier but couldn't find it.